Zone5/Transport/AccessToken.swift

```diff
@@ -6,7 +6,10 @@ public protocol AccessToken: Codable, CustomStringConvertible, CustomDebugString
 	/// The string value of the token.
 	var rawValue: String { get }
 	var tokenExp: Int? { get set }
+	var expiresIn: Int? { get set }
```

Comment:
Possible - will it encode/decode identically to the int?

Reply:
Should do (you can [check the source](https://github.com/Zone5-Cloud/z5-sdk-swift/blob/d9b84a90c9e3dd18ab37a79a9f7b1569f4f56929/Zone5/Search/Milliseconds.swift)). I specifically set it up to improve the interplay with Date while still allowing arithmetic operators and the like (`Date` typically uses `TimeInterval` for timestamp conversions, but because that's seconds (with microsecond accuracy because it's a `Double`) it doesn't fit the needs we had for other places that use millisecond values.